docs/reference/search-application/apis/search-application-render-query.asciidoc

```diff
@@ -27,19 +29,29 @@ Requires read privileges on the backing alias of the search application.
 
 `params`::
 (Optional, map of strings to objects)
-Query parameters specific to this request, which will override any defaults specified in the template.
+Query parameters used to generate the Elasticsearch query from the search template associated with the search
+application.
+If a parameter used in the search template is not specified in `params`, the parameter's default value will be used.
+
+[NOTE]
+====
+The search application can be configured to validate search template parameters.
+See the `dictionary` parameter in the <<put-search-application, put search application>> API for more information.
+====
 
 [[search-application-render-query-response-codes]]
 ==== {api-response-codes-title}
 
+`400`::
+Invalid parameter passed to search template.
```

Comment:
Should we add more information on invalid parameters, e.g. types different from what is defined, not defined in dictionary, etc?

Reply:
I will add more info on what "invalid parameter" means in this context